modules/stanford_news_extras/stanford_news_extras.module

```diff
@@ -5,3 +5,15 @@
  */
 
 include_once 'stanford_news_extras.features.inc';
+
+/*
+ * Implementation of hook_form_alter()
+ */
+function stanford_news_extras_form_alter(&$form, &$form_state, $form_id) {
```

Comment:
It is slightly less overhead to use hook_form_FORM_ID_alter rather than the raw hook_form_alter as it will be evaluated on every form.